src/auditor/dynamic_linkage.jl

```diff
@@ -39,7 +64,9 @@ function platform_for_object(oh::ObjectHandle)
             end
         end
 
-        if oh.ei.osabi == ELF.ELFOSABI_LINUX || oh.ei.osabi == ELF.ELFOSABI_NONE
+        if oh.ei.osabi == ELF.ELFOSABI_NONE
```

Comment:
Should we just delete `platform_for_object` since it appears to be unused?

Reply:
Sure, I can if you'd like me to.